### ISSUE TITLE
plugin_hyc_tenant breaks if inactive tenant does not have values

### ISSUE THREAD

**Issue by mstud** [metricq/metricq-source-ipmi#13]
Sometimes it can happen that the currently inactive tenant does not provide values, e.g.
```
77  | HYB_1 Tenant    | Chassis                  | N/A        | N/A   | 'OK'
78  | HYB_2 Tenant    | Chassis                  | N/A        | N/A   | 'Device Enabled'
93  | HYB_1 Temp. 0   | Temperature              | N/A        | C     | N/A
94  | HYB_2 Temp. 0   | Temperature              | 41.60      | C     | 'OK'
```
In this example, even though only the HYB_2 temperature is of interest because that's the active tenant, no values will be provided because NEEDED_SENSORS is not satisfied.
Fix: only check for the availability of the tenant sensors and the one active sensor. The fourth sensor should be irrelevant.

**Comment by mstud** [comment]
closed by 57479f060c88bd2dc01aab4dc5707fd2c799d5a5

**Comment by mstud** [comment]
or maybe not.

**Comment by mstud** [comment]
fixed in 90c8f7448a6a76ea7ebf91c5e7a65523f8bf8351 for real now
